### ISSUE TITLE
Add a way to configure default `sort` and `sortBy`

### ISSUE THREAD

**Issue by okapies** [aminalaee/sqladmin#189]
### Checklist

- [X] There are no similar issues or pull requests for this yet.

### Is your feature related to a problem? Please describe.

It is useful if we can configure default `sort` and `sortBy` in `ModelAdmin` (e.g. `sortBy="created_at"` and `sort="desc"` for log tables).

### Describe the solution you would like.

```python
class Log(Base):
    __tablename__ = "logs"

    id = Column(BigInteger, primary_key=True)
    created_at = Column(DateTime)


class LogAdmin(ModelAdmin, model=Log):
    sortBy = "created_at"
    sort = "desc"
```

### Describe alternatives you considered

I think it is useful if there is a way to specify custom `Statement` directly (like `get_queryset` in django-admin) because it also adds functionality to optimize the query:

```python
class LogAdmin(ModelAdmin, model=Log):
    def get_statement(self, request):
        ...
```

Note that a pain point of Django's implementation is that `get_queryset` is shared both in `SELECT` and `INSERT/UPDATE/DELETE`. There should be a cleaner way to do it.

**Comment by aminalaee** [comment]
There's `column_sortable_list`:

https://github.com/aminalaee/sqladmin/blob/5d2e125b660e295174a2440f9a13e7a8255e162c/sqladmin/models.py#L270-L279

It's missing documentation with some other configs, which I've added a PR to update.

**Comment by okapies** [comment]
Thanks, but it seems it just specifies sortable columns on the UI. The feature I mentioned is *default* sort order when opening the list page without parameters.

**Comment by aminalaee** [comment]
Yeah there's another config in Flask-Admin: https://flask-admin.readthedocs.io/en/latest/api/mod_model/?highlight=column_default_sort#flask_admin.model.BaseModelView.column_default_sort
I've created a PR to add this. Thanks for the suggestion :+1
